Dockerfile

```diff
@@ -24,7 +24,7 @@ ADD Gemfile* *.gemspec /home/app/src/
 RUN bundle install --frozen
 
 ADD package.json yarn.lock /home/app/src/
-RUN yarn install
+RUN yarn install --check-files
```

Comment:
This won't have any impact on running `yarn install` outside of the image build, since they are two separate installations. This bakes dependencies into the image, while executing via `docker-compose run` uses a volume.

Reply:
Ok - cool. Thank you! I will remove it.